hapi-loader/src/main/java/com/arkhn/hapi/loader/RedisCounterProperties.java

```diff
@@ -0,0 +1,39 @@
+package com.arkhn.hapi.loader;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@ConfigurationProperties(prefix = "spring.redis")
+@Configuration
+@EnableConfigurationProperties
+public class RedisCounterProperties {
+
+    private String host = "river-redis";
+    private Integer port = 6379;
+    private Integer db_index = 2;
```

Comment:
shouldn't this live in the conf file ?

Reply:
Which conf file?

Reply:
application.yaml ?

Reply:
Ah yes, I put them in the deployment template but not here 👍 
I think I still need some default value, though (don't know if you wanted me to remove them)

Reply:
yeah I kinda wanted these to be remove, feels like it leaks details from our deployment config. you sure you need the defaults ? I think it's fine if it fails if these are not defined in application.yaml